src/browser/standalone.ts

```diff
@@ -0,0 +1,128 @@
+import { Config } from "../config";
+import { NewBrowser } from "./new-browser";
+import { ExistingBrowser } from "./existing-browser";
+import { Calibrator } from "./calibrator";
+import { AsyncEmitter } from "../events";
+import { BrowserName, type W3CBrowserName } from "./types";
+import { getNormalizedBrowserName } from "../utils/browser";
+import { LOCAL_GRID_URL } from "../constants/config";
+import { WebdriverPool } from "../browser-pool/webdriver-pool";
+import RuntimeConfig from "../config/runtime-config";
+import type { CommonConfig } from "../config/types";
+
+export type StandaloneBrowserOptions = Pick<
+    CommonConfig,
+    | "automationProtocol"
+    | "desiredCapabilities"
+    | "gridUrl"
+    | "baseUrl"
+    | "httpTimeout"
+    | "pageLoadTimeout"
+    | "waitTimeout"
+    | "waitInterval"
+    | "headless"
+    | "prepareBrowser"
+    | "windowSize"
+    | "orientation"
+    | "headers"
+    | "strictSSL"
+    | "user"
+    | "key"
+    | "system"
+>;
+
+export type StandaloneBrowserOptionsInput = Partial<StandaloneBrowserOptions>;
```

Comment:
I can't just specify this option:
```
{
  system: {debug: false}
}
```

Looks like we should make partial nested options